third_party/airflow/tox.ini

```diff
@@ -0,0 +1,73 @@
+[tox]
+isolated_build = true
+skipsdist = true
+envlist =
+    format
+    py39
+
+[testenv]
+deps = 
+    poetry
+whitelist_externals = 
+    poetry 
+    bash
+commands =
+    poetry install -v
+    poetry run pytest tests/unit
+    # This executes the dag files in examples but really only checks for imports and python errors
+    bash -c "find examples/ -maxdepth 1 -type f | xargs poetry run python3"
```

Comment:
nit: if you added a `-name *.py`, you might save someone an annoying problem in the future :)